main.coffee

```diff
@@ -1,6 +1,7 @@
 dateFormat = require './date-format'
 
-module.exports = (options) -> new Logger options
+create = (options) -> new Logger options
```

Comment:
The name `create` is misleading. We don't understand why a console is attached to it.